### PR TITLE
[#531700912] Renamed Capacity endpoints

### PR DESCRIPTION
**Pull request by ShabiniRajadas**
### Description
Renamed endpoints of capacity management,

**/locations/{location_id}/capacities** - the endpoint the get the current number of signins, and invites expected soon,
**/locations/{location_id}/capacity_forecasts**- the endpoint that would give the data on invites expected over time 
renamed _hours_to_calculate_ param to _hours_to_forecast_

Added new endpoint - locations/{location_id}

### Changelog
- Write which paths/models/etc. are being modified so people know what to pay attention to

### CODE REVIEW

**Comment by fernandojinzenji** [comment]
Don't forget to bump the version, so a new SDK can be generated!

**Comment by prescottie** [comment]
unfortunately i think you'll have to bump the version for this as well... and since we are making a change in here couldn't we add a show endpoint for locations

**Comment by tractionguest-integrations** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  Remember to use the `Squash and Merge` button to merge!
    Always remember, your final commit message needs to start with a wrike ticket like, `[#1234] Your commit message`.
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  Associated Wrike Tickets:
<a href='https://www.wrike.com/open.htm?id=531700912' target="_blank">#531700912</a>
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  <a href="https://stoplight.io/p/docs/gh/tractionguest/guest-api/openapi.yml?srn=gh/tractionguest/guest-api/openapi.yml&group=capacity-endpoint-changes" target=_blank>View docs for this page</a>
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against eaf81d8d66a04b938f136c97f6103800c8b09328
</p>

File: openapi.yml [hunk 1]
```diff
@@ -3506,6 +3506,45 @@ paths:
...
+            application/json:
+              schema:
+                $ref: ./models/ErrorsList.v1.yaml
+        '401':
+          description: "You don't have permission to view this Location"
+        '403':
+          description: You do not have permission for this action
+        '404':
+          description: The Location does not exist
+      operationId: GetLocation
```
Comment (stevenallen05) [inline openapi.yml:3546]: Operation IDs should start with a lower-case letter. I know it seems trivial, but it helps a lot for the SDK generation.

```suggestion
      operationId: getLocation
```

**Pull request merged**